GenderPayGap.WebUI/Views/GOVUK/Layouts/template.cshtml

```diff
@@ -215,6 +214,8 @@
 <!--end footer-->
 <div id="global-app-error" class="app-error hidden"></div>
 <script src="@(ViewBag.assetsPath)/javascripts/govuk-template.js?0.13.0" type="text/javascript"></script>
+<script src="@(StaticAssetsVersioningHelper.GetAppJsFilename())"></script>
+<script>window.GOVUKFrontend.initAll()</script>

```

Comment:
These changes should mean that all the old templates will load the new css and js files

Reply:
I've also dropped this because I've seen you've already added this in